### DISCUSSION TITLE
Manuall cell placing with odb

### DISCUSSION THREAD

**Discussion by OuDret** [The-OpenROAD-Project/OpenROAD#4764] [Q&A] [answered]
Hello, 

Could you help me figuring out how to add and place some cell through command line to specific coords? 

Thanks

**Comment by maliberty** [comment, marked as answer]
By 'add' do you mean create new instances and connect them?  In general you'll have to use odb scripting for this.  https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/cts/test/array.tcl creates and places instances along with other operations.

**Reply by OuDret** [reply]
Thank you
